server/controller.js

```diff
@@ -83,6 +83,30 @@ module.exports = {
       .catch(err => {
         res.status(404).send(err);
       });
+  },
+
+  createUser: (req, res) => {
+    let newUser = {
+      _id: req.body._id,
+      username: req.body.username,
+      password: req.body.password,
+      email: req.body.email,
+      name: req.body.name,
+      address: {
+        street: req.body.address.street,
+        // street2: req.body.newUser.street2 || null,
+        city: req.body.address.city,
+        state: req.body.address.state,
+        zipcode: req.body.address.zipcode
+      }
+    };
+    model.addUser(newUser)
+      .then(() => {
+        res.sendStatus(201);
+      })
+      .catch(err => {
+        res.status(404).send(err);
+      });
   }
 };
```

Comment:
The createUser function looks almost the same as the addNewUser function I have for my component so we may be able to consolidate these.